sdks/java/nexmark/src/test/java/org/apache/beam/sdk/nexmark/model/sql/RowSizeTest.java

```diff
@@ -42,20 +42,20 @@
  */
 public class RowSizeTest {
 
-  private static final Schema ROW_TYPE = RowSqlTypes.builder()
-      .withTinyIntField("f_tinyint")
-      .withSmallIntField("f_smallint")
-      .withIntegerField("f_int")
-      .withBigIntField("f_bigint")
-      .withFloatField("f_float")
-      .withDoubleField("f_double")
-      .withDecimalField("f_decimal")
-      .withBooleanField("f_boolean")
-      .withTimeField("f_time")
-      .withDateField("f_date")
-      .withTimestampField("f_timestamp")
-      .withCharField("f_char")
-      .withVarcharField("f_varchar")
+  private static final Schema ROW_TYPE = Schema.builder()
+      .addByteField("f_tinyint")
+      .addInt16Field("f_smallint")
+      .addInt32Field("f_int")
+      .addInt64Field("f_bigint")
+      .addFloatField("f_float")
+      .addDoubleField("f_double")
+      .addDecimalField("f_decimal")
+      .addBooleanField("f_boolean")
+      .addField(Schema.Field.of("f_time", CalciteUtils.TIME))
+      .addField(Schema.Field.of("f_date", CalciteUtils.DATE))
+      .addDateTimeField("f_timestamp")
+      .addField(Schema.Field.of("f_char", CalciteUtils.CHAR))
+      .addField(Schema.Field.of("f_varchar", CalciteUtils.VARCHAR))
```

Comment:
I wonder if we should have a `.addField(name, type)` overload, maybe along with `.addNullableField()`